src/modules/common/components/Selector.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Link } from 'redux-little-router'
+import cx from 'classnames'
+
+import style from './Selector.css'
+import SelectorItemModel from '../models/SelectorItemModel'
+
+/**
+ * Displays a Selector showing different items
+ */
+class Selector extends React.Component {
+  static propTypes = {
+    verticalLayout: PropTypes.bool,
+    closeDropDownCallback: PropTypes.func,
+    items: PropTypes.arrayOf(PropTypes.instanceOf(SelectorItemModel)).isRequired,
+    active: PropTypes.string.isRequired
```

Comment:
We really need some docu for prop-types. "active" sounds like a boolean

Reply:
meeh :/

Reply:
I'd also like that we have the each SelectorItemModel has an active property, so we could have multiple items active. (For example: Use Selector as a Filter where multiple filter are active)

Reply:
@Schedulaar Dismissed.